README.md

```diff
@@ -20,8 +20,29 @@ Go and [register a new OAuth application](https://github.com/settings/applicatio
 
 Once you've created the application, you should be given the client id and the client secret. **Make a note of these**.
 
+Additionally, if the intention is to run the GitHub repository poller (`npm run poll-repositories`), a API token should be acquired so the API calls are submitted to an acceptable rate-limit (5k req/s instead of 60 req/s for anonymous access)::
+
+  curl -s -u '<GH_USER>' https://api.github.com/authorizations \
+    -H 'Content-Type: application/json' \
+    -H 'X-GitHub-OTP: <GH 2FA CODE IF ENABLED>' \
+    -d '{"client_id": "c54ee622a3fa9622701f", \
+         "client_secret": "2a6010d06c7c90b1fcf6518d72eac0bab64cf35b", \
```

Comment:
Hopefully this is not a real client secret (or at least not one we don't want leaked).
It would be good not to have specific ids in README and replace them with some <PLACEHOLDERS>.